### ISSUE TITLE
Same code, different animations

### ISSUE THREAD

**Issue by ghost** [ropenscilabs/learngganimate#27]
In the "Animate and renderer options" section of README, the box plots move horizontally. Yet, using the same code elsewhere, say in the "ease_ aes" section,  they don't. Do you know why?

**Comment by will-r-chase** [comment]
no idea, I'll look into it, maybe I made a mistake? I just copied and pasted the code from the "transition_states" tutorial

**Comment by ghost** [comment]
I recently updated versions of gganimate and dependencies,  and I noticed I too had the same problem - the horizontal movement of the boxplots - and other issues to do with placement of bars in bar plots. However, after a more recent update, the problems disappeared. Possibly, the problem was related to [this gganimate issue](https://github.com/thomasp85/gganimate/issues/217)

**Comment by adam-gruer** [comment]
Hey @SandyMuspratt great question, I just ran @will-r-chase code and the sideways movement is gone.  I'll re knit the animate doc and push up to the repo.   As you suggested  I am guessing that it has to do with what versions of `gganimate`, `tweenr` and `transformr` and other dependencies, that a user has installed.  This is related to parts of #32 and #29 and the fact that Thomas is still developing all of this.  I have now installed the CRAN versions of `tweenr` and `transformr` and I think things are all ok.
